tests/bluetooth/init/testcase.yaml

```diff
@@ -77,18 +77,35 @@ tests:
     extra_args: CONF_FILE=prj_ctlr.conf
     platform_allow: nrf52840dk_nrf52840 nrf52dk_nrf52832
       nrf51dk_nrf51422 rv32m1_vega_ri5cy
+    integration_platforms:
```

Comment:
do we really need to add all of them? Can't we just add one that can provide the coverage? This is exactly why we have integration_platforms...

Reply:
I know we need more than one as we keep having failures in the nightly from time to time since we aren't getting coverage here.  Now which set of the 4 do we need I don't exactly know.  One of the Nordic guys probably knows best what the difference between the 3 nordic platforms and what coverage we get.

Reply:
I have requested change, selectively removed some boards so that the coverage is sufficient.